challenges/functions.js

```diff
@@ -6,28 +6,40 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+function consume(a,b,cb){
+  console.log(cb(a, b));
```

Comment:
**You need to return** `the function return the callback with the two parameters that you created`